### DISCUSSION TITLE
Overclock Labs Acquires Cloudmos To Expand the Akash Supercloud: Proposal

### DISCUSSION THREAD

**Discussion by brewsterdrinkwater** [akash-network/community#286] [Governance Proposals]
# Overclock Labs Acquires Cloudmos To Expand the Akash Supercloud

submitted by: Tyler | Core Team

## Takeaways:

- Overclock Labs (OCL) and Cloudmos are joining forces to co-develop open-source deployment tools for Akash Network.
- Cloudmos Deploy will become fully open-source.
- **OCL will contribute $100,000 to the acquisition and will also request $200,000 from the Community Pool.**
- OCL and Cloudmos will work together closely for a minimum of 6 months to co-develop Akash Console and Cloudmos Deploy.

## Mission: 

Overclock Labs is excited to join forces with the Cloudmos team to co-develop products with the goal of expanding Akash Network. This collaborative acquisition is the result of months of work between both teams to come to an agreement on terms that further our shared goal of developing great open-source products for Akash Network.

## Overview: 

The four points below provide an overview of the acquisition terms, which are designed to ensure a smooth transition and support successful collaboration between Overclock Labs and Cloudmos.

#### A Commitment to Open-Source

As a key part of the acquisition, Cloudmos will open-source its entire codebase. This will align the Cloudmos suite of products with Akash Console, which is already fully open-source. By sharing code and opening the development process, developers from around the world will be able to contribute directly to the growth of the Akash ecosystem.

#### Collaborative Development

Overclock Labs and the Cloudmos team will work together closely for a minimum of 6 months to continue developing Cloudmos and Akash Console. During this period, we will actively collaborate, and share code, to eliminate redundancy and increase the speed of development. This massively expands the potential user base for Akash. By collaborating closely, Cloudmos Deploy and Akash Console will be able to create a [cohesive strategy](https://github.com/akash-network/community/blob/5bc4d0af07dbb4ea955e7ebed19393fabe69891b/product-strategy/cloudmos-console-product-strategy.md) that addresses distinct user segments both within web3 and in the broader development community outside of web3.

#### Community Pool Funding
**Overclock Labs is requesting $200,000 in AKT** (Akash Network's native token) from the Community Pool. This funding will be utilized to support the continued growth, and development of Cloudmos along with fully open-sourcing the Cloudmos codebase. The AKT granted will vest over a 6-month period. [Please review the Cloudmos Deploy and Akash Console product strategy to learn more.](https://github.com/akash-network/community/blob/5bc4d0af07dbb4ea955e7ebed19393fabe69891b/product-strategy/cloudmos-console-product-strategy.md)

#### Contribution from Overclock Labs
**Overclock Labs will contribute $100,000 in AKT alongside the funding from the Community Pool**. This contribution demonstrates our commitment to the success of Cloudmos and Akash Network. The AKT contributed will vest over a 6-month period.

## Conclusion

Everything related to Akash is fully open-source. This applies to the code that powers the network, but it also extends to the processes for governance and management. Contributors engage with Akash through a unique structure of working groups that meet publicly. Anyone from around the world can attend. This radically open structure allows for a level of collaboration that is not possible in closed-source development. 

Among this global community of developers, Cloudmos has emerged as a standout contributor. Their flagship application, Cloudmos Deploy, is one of the most widely used tools in the Akash ecosystem. They have also moved quickly to ship popular tools like Cloudmos Alerts and Akashlytics.

This collaboration represents a win-win-win scenario for Akash Network’s community, Overclock Labs, and Cloudmos. By joining forces, Cloudmos will continue to make it easier and more intuitive to interact with Akash.

The primary goal of this acquisition is to ultimately deliver more value to our community as we move towards cloud parity. With a shared codebase, we will be able to improve UX, ship new features quickly, and reduce duplicate work. Together, we will accelerate the development of the world’s first operational Supercloud.

**Comment by Aihonaken** [comment]
Great new guys ! Happy for the Overclock and Cloudmos. I have a few questions though : 
- As far as i understand, overclock and community are giving to the cloudmos team 300k for 6 months. What's going to happen after that? Who is going to be developing the cloudmos after all the money will be paid in full? 
- How does the cloudmos team plan to sell the tokens? i'm concerning about the price fluctuations if they are planning to sell it by a market sell order. 
- Cloudmos had an attempt to implement the paid features into the platform? Will it's still be a thing? And if yes, will the money from paid features go to the community pool? 


Again congratulations !

**Reply by baktun14** [reply]
Hey Andrew,

> As far as i understand, overclock and community are giving to the cloudmos team 300k for 6 months. What's going to happen after that? Who is going to be developing the cloudmos after all the money will be paid in full?

The working contract is for 6 months currently, but there's a possibility it continues after that. Also, in the next 6 months we will work on making Cloudmos documented to make sure, in the event we move on to other things, that it's easy to pick up the development and to run.

> How does the cloudmos team plan to sell the tokens? i'm concerning about the price fluctuations if they are planning to sell it by a market sell order.

The payments are vested over the next 6 months and we will try to sell the tokens with the least impact possible. We also plan on keeping a portion of the tokens, but despite this, there's well enough liquidity to support us and should have minimal impact on the price.

> Cloudmos had an attempt to implement the paid features into the platform? Will it's still be a thing? And if yes, will the money from paid features go to the community pool?

I think this will be up to Overclock Labs to decide if they want to implement paid features.

Thanks!

**Comment by zJuuu** [comment]
I'm a bit irritated, it was published that Cloudmos has already been acquired by Overclock but this proposal requests the funds for that?

Also, the wording is confusing. Who acquired Cloudmos? Overclock or the community, if 2/3 of the money comes from the community.

How was the value of Cloudmos and the amount for the next 6 months determined? 

Why aquire cloudmos if the community could give them a grant with a aligned roadmap?

**Reply by gosuri** [reply]
I'm sorry you feel "irritated" by months of efforts of Overclock and Cloudmos teams to keep continue their contributions and find a sustainable path forward in light of the community rejecting their grant [proposal](https://www.mintscan.io/akash/proposals/185).

Overclock will acquire the legal entity of Cloudmos to open source all their work so far to the community.

It should be clear if you spend more than 5 seconds reading past the title. Where is the confusion?

**Reply by zJuuu** [reply]
Thank you for your reply. These efforts are not apparent to community members who can only gain insight based on this proposal and the attached product strategy.

However, I am convinced that Overclock and Cloudmos have put a lot of effort into finding a solution and I am very grateful for that. There is no question that Cloudmos is the best tool for deploying to Akash and the open sourcing is incredibly valuable to the entire community.

I am also aware of the basic idea of the proposal but it reads differently to me and I am grateful that the questions have been answered.

**Comment by brewsterdrinkwater** [comment]
@zJuuu, Thanks for the feedback and we completely understand your concerns. 

Here are the answers to your questions, along with context on how these decisions were made. Let us know if we can clarify anything or offer more detail.

1. Overclock Labs is directly acquiring Cloudmos.
2. The community funding reflects the time and effort that the Cloudmos team will have to go through to fully open-source their entire codebase. They will also co-develop their suite of products alongside Overclock Labs for a minimum of 6 months.
3. The funds provided by Overclock are essentially a subsidy to the community pool. Overclock Labs will not retain **any** private IP from this acquisition. Everything will be open-source.
4. **_Beyond_** this acquisition funding, Overclock Labs will be compensating the Cloudmos team for a minimum of 6 months of development work to oversee the transition.

As mentioned in the original post, this acquisition will allow us to increase development speed and efficiency, which will result in better products and faster development cycles. All of this will happen through a transparent and open-source process that will ultimately benefit the entire Akash community.

**Reply by zJuuu** [reply]
Thanks a lot for clarifying

**Comment by JanPapaj-reload** [comment]
Greg, regarding your below statement, I have a question: 
I'm sorry you feel "irritated" by months of efforts of Overclock and Cloudmos teams to keep continue their contributions and find a sustainable path forward in light of the community rejecting their grant proposal.

1) 
Going forward, do you not see a key risk in this on-chain governance system, where pretty solid devs (allegedly bringing 58pct of "new business" through their deploy tool) get rejected so badly? How could that even have happened, prop 185?  Rejected by token holders not even reading who Cloudmos are, what they´ve built. Thing is, these bad strategic decisions cost time and capital. Web2 Corps cannot afford that. Startups even less so, given their runway is limited. Moving fast while staying lean and shipping is the name of the game. 

In the long run, this system of peeps taking what I call a "majority vote", then moving on with their lives (I do that too with most other chains), is a highway to hell. Don´t you guys see that? **Is there a way to revamp the gov system to make it more agile?** 
In web2 BoD decide what gets done. They know their shit and they make informed decisions. Like you guys at OCL probably. 
In Web3, however, none of that is present. Retail token holders make strategic decisions without any insight. That´s a huge systemic risk. 

Any gov upgrade options considered please, or none at all? 

2) 
Now that Cloudmost gets acquired by OCL (good decision! Bravo) taking it under the roof, albeit for the little funds left in the pool, I have a direct question on you: 
**When OCL eventually implements paid features, will OCL direct a portion of the fees back to the comm pool?**   I may be wrong but that may be the "irritation" that zJuuu was feeling? 

After all, what else would the value accrual for the network be once you start monetizing the cloudmos deploy "gate"? (product development and open-source lib aside, of course)

3) 
That brings me to the last question? Do you feel a close-to-empty Community Pool is a problem in the long run, and if so, how should it be refilled?

Thank you for your precious time!

**Reply by gosuri** [reply]
> Going forward, do you not see a key risk in this on-chain governance system, where pretty solid devs (allegedly bringing 58pct of "new business" through their deploy tool) get rejected so badly? How could that even have happened, prop 185? Rejected by token holders not even reading who Cloudmos are, what they´ve built. Thing is, these bad strategic decisions cost time and capital. Web2 Corps cannot afford that. Startups even less so, given their runway is limited. Moving fast while staying lean and shipping is the name of the game.

> In the long run, this system of peeps taking what I call a "majority vote", then moving on with their lives (I do that too with most other chains), is a highway to hell. Don´t you guys see that? Is there a way to revamp the gov system to make it more agile?
In web2 BoD decide what gets done. They know their shit and they make informed decisions. Like you guys at OCL probably.
In Web3, however, none of that is present. Retail token holders make strategic decisions without any insight. That´s a huge systemic risk.

> Any gov upgrade options considered please, or none at all?


Companies building Akash unable to get funding is a systemic risk for Akash; [Prop 185](https://www.mintscan.io/akash/proposals/185) rejection proves this.

We proposed a Public Goods Pool with [AKT 2.0](https://github.com/gosuri/akt20/blob/main/README.md#public-goods-fund) to offset this risk to optimize funding for tools that enhance Akash Network.

> 2. Now that Cloudmost gets acquired by OCL (good decision! Bravo) taking it under the roof, albeit for the little funds left in the pool, I have a direct question on you:
> When OCL eventually implements paid features, will OCL direct a portion of the fees back to the comm pool? I may be wrong but that may be the "irritation" that zJuuu was feeling?

> After all, what else would the value accrual for the network be once you start monetizing the cloudmos deploy "gate"? (product development and open-source lib aside, of course)

A primary reason for OCL acquiring Cloudmos is to opensource their tools. OCL has no plans to implement paid features on Cloudmos deploy and explorer tools.

> 3. That brings me to the last question? Do you feel a close-to-empty Community Pool is a problem in the long run, and if so, how should it be refilled?

There is close to $1 million in the community pool; with the recent passing of [Prop 211](https://www.mintscan.io/akash/proposals/211), the Community pool is expected to grow to $2.8 million. AKT 2.0 has proposed additional pools to fund community efforts as well.


Thanks for questions, hope this helps.

**Reply by JanPapaj-reload** [reply]
Thank you for taking the time! ....Looking good. As always: All my support to the builders, not marketers right now. OCL taking Cloudmos under the wings, injecting and supporting "cloudmos boys", is a good decision. Always AKT and never different

**Comment by KamuelBob** [comment]
After having read the proposal and the discussion here, there are a number of irregularities with the messaging that are concerning.

As we read in the announcement [here](https://twitter.com/akashnet_/status/1686386296007344128) and [here](https://twitter.com/ovrclk_/status/1686384137756884992), Overclock Labs have acquired Cloudmos. This proposal then describes that "The funds provided by Overclock are essentially a subsidy to the community pool. Overclock Labs will not retain any private IP from this acquisition. Everything will be open-source."

While we support the end goal (Cloudmos being funded to open source their work), the wording in the proposal appears misleading. Should we not be agreeing whether the Akash Community Pool should be funding the open sourcing of Cloudmos or if indeed Cloudmos is being/has been acquired, should the proposal not be asking for the community fund to be paying Overclock Labs to open source their tooling which they have just acquired.

The proposal then goes on to describe the use of funds is for the acquisition, and seems to suggest that Overclock will be paying Cloudmos for the open sourcing work themselves: "Beyond this acquisition funding, Overclock Labs will be compensating the Cloudmos team for a minimum of 6 months of development work to oversee the transition."

There is also no description of the account that will receive the funds, is this controlled by Max/Cloudmos or Overclock Labs/a multi-sig?

It's also a little convenient that we have started using a 30 day moving average for price calculations (See Proposal [208](https://www.mintscan.io/akash/proposals/208), [207](https://www.mintscan.io/akash/proposals/207), [200](https://www.mintscan.io/akash/proposals/200), etc.) given recent price movements however that's not necessarily an issue as long as proposers don't pick and choose in their favor each time.

Are we just missing something or is it very confusing messaging? Looking forward to having the above clarified so that we can make an informed vote.

EDIT: It makes a little more sense when I realised that Overclock have acquired Cloudmos the tool, not the company/team behind it. Nevertheless some of the above points are still valid.

**Reply by hewi333** [reply]
Hi KamuelBob,

Thank you for taking the time to read the proposal and provide a detailed response. It’s important that validators keep up with governance so hats off to you for that. We are also glad to hear you support the end goal (Cloudmos being funded to open-source their work). I will attempt to clarify some nuances with this proposal.

We’ll start with the acquisition language, timing of the announcement, and timing of the governance proposal. Regardless of whether this passes, Overclock Labs has acquired Cloudmos and is working with them to open source the codebase. 

As of now, Overclock Labs has the financial burden to make this open-source. We are confident that this decision reflects our commitment towards building a [public cloud](https://akash.network/blog/public-cloud-is-a-public-utility/), our community-first approach, and our long term vision for Akash Network. If you feel that the community funds are not to be used to open-source Cloudmos, continue feature development, and make the analytics and indexer freely available to anyone — we would appreciate your thoughtful response.

The wallet address was not explicitly mentioned in the proposal description but you can always click the “JSON Data'' button in Mintscan to see the full details. Mintscan 2.0 has a ton of great changes but now on proposals it only shows the proposer address and not the recipient by default. This is something we will make sure to include in the proposal description moving forward to avoid any confusion. From the JSON, "recipient": "akash1v6c77xl46cfwcxcg30urygvpn8sv8rp6mwaluz". This address is controlled by Overclock Labs and will pay out Max/Cloudmos over 6 months. $200k/6 mo. = $33.3k/mo. Notice that these amounts are US dollar denominated and not AKT denominated. It is important to note that should there be any remaining funds leftover after the 6 months, those funds will be returned to the community pool.

We took a different approach from props 200, 207, 208 in requesting AKT at a moving average vs. spot. Those programs (Events & Community Support) are ongoing programs that are reported on and have quarterly proposals. Any surplus or deficit can be accounted for in subsequent quarterly proposals. Prop 216 is a one time purchase vested over 6 months. Given the volatility of AKT and the fact that any unspent AKT will be returned to the community pool, we believe this ensures: 

1. Cloudmos actually gets $200k of AKT and; 
2. Does it in a way that does not negatively impact price. In the event that the price drops below the moving average rate Overclock Labs requested, we would not submit another proposal, but would instead cover that deficit.

Since the start of 2023, Overclock Labs has been working to make all efforts open-source. Building in the open is something that Overclock Labs takes pride in. There are Special Interest Groups and Working Groups purpose-built around a plethora of topics and themes, that coordinate and meet via [Github](https://github.com/akash-network/community), Discord, and monthly calls that are all recorded and transcribed. These meetings and groups are conducted publicly, and anyone is welcome to attend.

**Reply by KamuelBob** [reply]
Hey heyhewi,

Thanks for the thoughtful reply. Yes, we take governance seriously and hold a pretty high expectation for proposers.

It is more clear now the scope of the proposal and although the on chain proposal can't be updated, it does link to this discussion will hopefully help those who had the same confusion as us. Also apologies for not reviewing this discussion prior to it being listed on chain, we need to keep a closer eye on the forum.

As a general recommendation, it might be worth proposers (anyone, not just Overclock Labs) pinging validators on the Discord and even asking the general community on Telegram/X for a review of the proposal prior to it being listed on chain. This might help improve the quality of proposals, we did this on Proposal 7 and it definitely helped.

Yes we can see the recipient account on chain, however it is good practice to describe the owner of the account and the payment terms in the proposal so thanks for clarifying that.

Edit: As a result we have voted YES on the proposal.

Thanks for all the hard work by you and the rest of the Overclock Labs team, we keep up the good work, we're proud to be part of the ecosystem. Keep building!

**Comment by MadEyeMartini** [comment]
Hi,

Personally not for this, I feel the proposal massively overvalues this acquisition.    

1. **Redundancy in Development:** Cloudmos/Akashlytics initially received funding to develop an open-source desktop tool. It's apparent that the value of the existing desktop app significantly undermines the value proposition of the new web app. If the web app does not come with markedly superior or differentiating features, then the proposal seems more like a duplication of efforts, potentially leading to inefficient use of resources.

2. **Exorbitant Acquisition Cost:** Allocating $300,000, with the community shouldering $200,000 of it, for the acquisition is substantial. To put it into perspective, if the web app's feature set isn't vastly different from the desktop version, why should the community bear such a high cost for what could potentially be achieved by building on the already existing desktop app?

 3. **High Compensation for Short-Term Commitment:** The proposal highlights a six-month commitment from Cloudmos, and while it's important to adequately compensate developers for their efforts, $300,000 for 6 months of work seems on the higher side, especially considering the overlapping functionalities between the existing desktop app and the new web app.

 4. **Community Pool Utilization:** Drawing $200,000 from the Community Pool is significant. By today's valuation, that's a sizable percentage of the pool's resources. It's essential to ensure that these funds are channeled towards projects that add undeniable value to the Akash ecosystem, rather than on projects with questionable returns on investment.

5. **Transparency & Detailed Breakdown Needed:** While it's highlighted that the acquisition cost reflects the time and effort to open-source the Cloudmos codebase, a more granular breakdown of these costs would help the community better understand and evaluate the worthiness of this expenditure.

6. **Potential for a Troublesome Precedent:** Approving such a proposal could inadvertently set a dangerous precedent, incentivizing developers to initially create closed-source tools on Akash. If the community is seen as willing to pay significant sums to open-source these tools later on, it could unintentionally encourage a model where developers build in a closed environment with the potential goal of a lucrative buyout. This goes against the spirit of open-source development, which thrives on collaboration, transparency, and shared progress from the outset. It's paramount to ensure that the Akash ecosystem remains a place where open-source development is the norm, not something that needs to be financially incentivized after the fact. By passing this proposal, we might inadvertently be signaling to developers that there is a financial reward for initially withholding code from the community, only to release it later for a fee. This could bottleneck the network and undermine the principles that make the Akash community vibrant and innovative.

Point 6 tl;dr - Imagine this. This proposal is passed. Developers now know that they can build closed source tools, encourage reliance on them by the community, once there is enough reliance they can withhold development and force the community to "acquire" them.

Sadly not staking so my vote amounts for little.

**Reply by JanPapaj-reload** [reply]
Ad 2) 
With all due respct to your other concerns (some of them rational), you are suggesting skimping on one of the most productive external teams building on Akash. The team of two (!) developers composing all that work behind Cloudmos, has shown talent, and self-steering drive. You will hardly find that level of passion+skills elsewhere! 

These folks (ofc, credit also goes to Praetor team, plus Quasarch, Vixello more recently, and others) have done what Akash protocol is today. And however I also see 300k in USD as a lot of money, I would not even try to imply that OCL are trying to tunnel "their own" chain :)

ALL THE SUPPORT TO (PROVEN) BUILDERS! 
(Funny that you folks did not have these objections when recent "marketing budget" props passed, drawing huge funds from the pool. Marketing budget in a deep bear is nonsense and waste, you will agree with me. "Cloud parity" is still a looong way from here and when BTC halving brings the next bull, the Akash protocol should better be way closer to it. Now simply is a build time!) 

Other than that, if I understood the reasoning behind the prop well, the payout is mostly for a 6-month long job of making  Cloudmos´s code fully open-source, plus compensation for acquiring their code. 
As a non-dev, I have no way of telling whether that is too much or not. But I don´t see a reason why anyone, let alone OCL key management, would wish to tunnel their own brain child, thus draw bad publicity and in effect torpedo the project :) Sorry, but that doesn´t make sense. 

The 300k price tag, for me, is a generous payout for an outstanding contribution and commitment, after the community, sadly, rejected Prop 185 with such a huge margin. Shameful. 

Most importantly: Always AKT! Peace

**Comment by SophoclesSummer** [comment]
This proposal was approved on 8/15/2023 by a 94% majority. 
https://www.mintscan.io/akash/proposals/216
